examples/src/main/java/tachyon/examples/Performance.java

```diff
@@ -172,7 +171,7 @@ public void memoryCopyParition() throws IOException {
     @Override
     public void run() {
       try {
-        memoryCopyParition();
+        memoryCopyPartition();
```

Comment:
Thanks for the fix here. For future reference, try not to make additional changes unrelated to the ticket in the same PR.